lua/neogit/lib/popup/lib.lua

```diff
@@ -79,16 +79,26 @@ local function draw_popup(popup)
 
   for i=1,actions_grid_height do
     local result = " "
-    for _, col in pairs(columns) do
+    for index, col in pairs(columns) do
       local item = col.items[i]
+
+      local next_col = columns[index + 1]
+      local has_neighbour = (next_col and col.items)
+        and (item and next_col.items[i])
+        or false
```

Comment:
This looks like you wanted to build a ternary, but it ends up just being a boolean calculation.
Should be equivalent to just
```lua
next_col and col.items and item and next_col.items[i]
```
right?

Reply:
Yep, that's right forgot to condense it down from an earlier iteration